.eslintrc.json

```
@@ -1,4 +1,78 @@
 {
-  "extends": ["@flysas-tech/eslint-config-adg/node"],
-  "root": true
+  "extends": [
+    "standard",
+    "plugin:vitest/recommended",
+    "plugin:import/errors",
+    "plugin:jsdoc/recommended",
```

I recommend to change this the ts flavour as we are using jsdoc for types
```suggestion
    "plugin:jsdoc/recommended-typescript-flavor",
```
or this if we want errors instead of warnings
```suggestion
    "plugin:jsdoc/recommended-typescript-flavor-error",
```